src/services/userService.js

```diff
@@ -0,0 +1,41 @@
+const { generateUniqueId } = require('../utils/getRandomId');
+
+let users = [];
+
+const start = () => {
+  users = [];
+};
+
+const getUsers = () => {
+  return users;
+};
+
+const getUserById = (id) => {
+  return users.find((user) => user.id === id) || null;
+};
+
+const create = (name) => {
+  const newUser = {
+    id: generateUniqueId(),
+    name,
+  };
+
+  users.push(newUser);
+
+  return newUser;
+};
+
+const remove = (id) => {
+  users = users.filter((user) => user.id !== id);
```

Comment:
The `remove` function does not return any value. Consider returning a confirmation or the removed user for better feedback and error handling.